src/main.cpp

```diff
@@ -1274,6 +1274,12 @@ extern meshtastic_DeviceMetadata getDeviceMetadata()
     deviceMetadata.excluded_modules |= meshtastic_ExcludedModules_AMBIENTLIGHTING_CONFIG;
 #endif
 
+#if defined(ARCH_NRF52) && !defined(NRF52_USE_JSON) // nrf52 doesn't have network unless it's a RAK ethernet gateway currently
```

Comment:
Right now the RAK ethernet module/mode is active in these variants: monteops_hw1, rak4631, rak4631_eth_gw, rak11310 and rak_wismeshtap

I think we can drop it from wismeshtap and rak4631 proper (which would be a separate PR and release notes), but the others have to stay. Especially RAK11310 which is the RP2040 core.